x-pack/solutions/observability/plugins/slo/public/pages/slos/components/slo_list.tsx

```diff
@@ -82,6 +82,9 @@ export function SloList() {
   usePageReady({
     isReady: !isLoading && sloList !== undefined,
     isRefreshing: isLoading,
+    meta: {
+      description: '[ttfmp_services] The SLOs list has loaded and SLO data is present',
```

Comment:
suggestion: ttfmp_slo_list

❓ What does it mean that the SLOs list and data are loaded? Is the data related to SLO list or another component on the page?

Reply:
I based this off the way the `isReady` prop is computed. All we know at this point is that the loading procedure finished and the sloList is not equal to `undefined`.

We can make this a bit more helpful with a message like:

```
[ttfmp_slos] The SLOs list has finished loading. The SLO list is displaying all content it was able to fetch.
```

Or, we can compute this by including the number of SLOs that are in the list. Would that be ok? I am also open to suggestion. 😄